salesforce/10.10/modules/ROOT/pages/salesforce-connector-processing-events.adoc

```diff
@@ -9,88 +9,88 @@ You can use Anypoint Connector for Salesforce (Salesforce Connector) as an inbou
+To process events with Salesforce Connector, ensure that you have:
+
+* Access to the Salesforce target resource
+* The required Salesforce Streaming API permissions enabled in your organization
 
-* You must have access to the Salesforce target resource.
-* Before you can receive events for Salesforce changes associated with a topic, you must create the topic, if it does not exist.
-* You must have the required Salesforce Streaming API permissions enabled in your organization.
+Before you can receive events for Salesforce changes associated with a topic, create a topic using Salesforce Connector if the topic doesn't already exist.
 
 [[objectstoreusage]]
-== Object Store Usage
+== Object Store Usage in Events
 
-Both Salesforce Connector and Mule use an object store to persist data for features such as automatic message replay and message redelivery:
+Both Salesforce Connector and Mule runtime engine (Mule) use an object store to persist data for features such as automatic message replay and message redelivery:
 
 * The object store included in Mule for on-premises deployments of Mule apps has no transaction limits.
 * The free version of Object Store for CloudHub deployments of Mule apps has a limit of 10 transactions per second.
 
-For more information about object store versions, see https://docs.mulesoft.com/object-store/#object-store-notes[Object Store Notes].
-
+For more information about Object Store versions, see https://docs.mulesoft.com/object-store/#object-store-notes[Object Store Notes].
 
 === Replay Topic and Replay Channel Listener Operations
 
-The *Replay topic* and *Replay channel listener* operations have the option to continue from the last replay ID they received before restarting the application.
+The *Replay topic* and *Replay channel listener* operations can continue from the last replay ID they received before restarting the application.
 
-When a Mule app starts for the first time, the connector creates an object store that is used to save data related to messages that were processed successfully or that failed:
+When a Mule app starts for the first time, the connector creates an object store that saves data:
 
-* For each message that was processed successfully, the information stored consists of a replay ID associated with the message.
-* For each message that failed, the information stored consists of a number that represents the lowest replay ID for which the processing failed.
+* For each message that processes successfully, the connector stores the replay ID associated with the message.
+* For each message that fails, the connector stores a number that represents the lowest replay ID for which the processing failed.
 
-These two structures ensure that no message is processed twice and that when the application is restarted, it can reprocess the failed messages.
+These two structures ensure that no message is processed twice and that when the application restarts, it can reprocess the failed messages.
```

Comment:
+ These information is stored in the object store for 72 hours after which they are automatically removed.

Reply:
I agree this is important, might be worth adding it to the doc.